### ISSUE TITLE
TypeError - not all arguments converted during string formatting

### ISSUE THREAD

**Issue by ernstae** [Qumulo/capacity-by-user#3]
Maybe this tool requires that the cluster be running 2.5? 
Qumulo QC208 cluster is running 2.1.17

Regardless, I wanted to report this error 

```➜  capacity_by_user git:(master) python capacity_by_user.py -U admin -P redacted -C cluster-name -c 5 -D 10.01 /directory
Traceback (most recent call last):
  File "capacity_by_user.py", line 263, in <module>
    main(process_command_line(sys.argv[1:]))
  File "capacity_by_user.py", line 207, in main
    print "Total: %s" % (format_capacity(args.samples))
  File "capacity_by_user.py", line 198, in format_capacity
    return "$0.02f/month" % (to_dollars(0))
TypeError: not all arguments converted during string formatting```

Python version:
Python 2.7.13

**Comment by justinmahood** [comment]
Hey @ernstae , no issue with version mis-matches, just a bug (if you use -D without -i as well in the args).   Looks like @pgodman fixed it in #4 , should be merged soon.

**Comment by TommyUnger** [comment]
This appears to be fixed now. We also handle more user types.
